docs/source/llms/deployments/index.rst

```diff
@@ -693,25 +693,25 @@ For specifying an API key, there are three options:
 
 .. _deployments_query:
 
-Querying the AI Gateway
-=======================
+Querying the Deployments Server
+===============================
 
-Once the MLflow AI Gateway server has been configured and started, it is ready to receive traffic from users.
+Once the MLflow Deployments Server has been configured and started, it is ready to receive traffic from users.
 
 .. _standard_deployments_parameters:
 
 Standard Query Parameters
 -------------------------
 
-The MLflow AI Gateway defines standard parameters for chat, completions, and embeddings that can be
-used when querying any route regardless of its provider. Each parameter has a standard range and
-default value. When querying a route with a particular provider, the MLflow AI Gateway automatically
+The MLflow Deployments Server defines standard parameters for chat, completions, and embeddings that can be
+used when querying any endpoint regardless of its provider. Each parameter has a standard range and
+default value. When querying an endpoint with a particular provider, the MLflow Deployments Server automatically
 scales parameter values according to the provider's value ranges for that parameter.
 
 Completions
 ~~~~~~~~~~~
 
-The standard parameters for completions routes with type ``llm/v1/completions`` are:
+The standard parameters for completions endpoints with type ``llm/v1/completions`` are:
```

Comment:
We might want to mention that we confirm to the OpenAI schema somewhere.

Reply:
Could be a nice section to add under `Querying the Deployments Server` 

``` rst
API Syntax Standards
----------------------

The MLflow Deployments Server endpoint interfaces conform to those of the standards established by OpenAI. Other providers input and output interfaces will be translated to the OpenAI specification in order to provide a single standardized API interface when using the MLflow Deployments Server. 
```

WDYT?

Reply:
Sounds good. I'll add this in a separate PR.